apps/foundation-app/src/app/design/molecules/image-gallery/image-gallery.component.scss

```diff
@@ -0,0 +1,59 @@
+@import '~scss/styles';
+
+// parent element must have a margin-bottom of 100vw;
+.image-gallery {
```

Comment:
So, I think ideally we'll want one main image, and a few smaller images. So, I think functionally, what I'd expect here would be a small set of images that I can click on to switch the main image out. So, I think this nicely fits into two separate components: 
1. `image-list`
2. `image-gallery`

An `image-gallery` is composed of a main `img` and an `image-list`. Selecting on an item in the `image-list` makes that specific `image` the main image, and also applies some kind of distinction that the specific image in the list is selected as the main image (could be a border, or a icon, etc).

Reply:
done. I made the distinction between selected and deselected images be a difference in opacity.